src/app/components/Results/ResultsList.jsx

```diff
@@ -147,7 +147,6 @@ class ResultsList extends React.Component {
               items={items}
               bibId={bibId}
               getRecord={this.getItemRecord}
-              id="search-result-item-table"
```

Comment:
Probably should pass `null` to <ItemTable> if we don't need the ID attribute from it.